[Docathon] Fix api_base_io_shuffle API Label

entired opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->
others
### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->
Docs
### Description
<!-- Describe what you’ve done -->
删掉了paddle.base.io.shuffle 到 is an alias，诶嘿
@ooooo-create @sunzhongkai588

entired committed:
modified:   python/paddle/reader/decorator.py

ooooo-create commented on python/paddle/reader/decorator.py:
```diff
@@ -126,7 +126,7 @@ def reader():
 def shuffle(reader, buf_size):
     """
-    paddle.base.io.shuffle ( :ref:`api_base_io_shuffle` ) is recommended to use,
+    paddle.base.io.shuffle ( :ref:`api_paddle_io_shuffle` ) is recommended to use,
```
这一段是说明了别称的问题，但是这个 api 现在并没有公开暴露出来，可以整段都删了叭～

entired replied:
整段删？具体是哪里到哪里啊≥﹏≤

ooooo-create replied:
第一句话叭，`paddle.base.io.shuffle` 到 `is an alias.`～

entired replied:
谢谢大佬QvQ

ooooo-create replied:
其实可以不用关这个 pr 哈，你在这个分支上做修改， commit ，然后 push 就可以了～，pr 里会显示你的 commit 的

entired replied:
好耶，现在commit改了吗？

ooooo-create replied:
yes

ooooo-create reviewed (commented):
nice work～ @sunzhongkai588

entired committed:
[Docathon] Fix api_base_io_shuffle API
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->
others
### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->
Docs
### Description
<!-- Describe what you’ve done -->
删除了paddle.base.io.shuffle 到 is an alias.
@ooooo-create @sunzhongkai588

ooooo-create reviewed (approved):
LGTM~

sunzhongkai588 reviewed (approved):
LGTM
